packages/synapse-interface/utils/hooks/useIsBridgeApproved.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react'
+import { zeroAddress } from 'viem'
+
+import { stringToBigInt } from '@/utils/bigint/format'
+import { BridgeQuote, Token } from '@/utils/types'
+
+export const useIsBridgeApproved = (
+  fromToken: Token | null,
+  fromChainId: number,
+  bridgeQuote: BridgeQuote | null,
+  debouncedFromValue: string
+) => {
```

Comment:
Could we have `fromToken, fromChainId, bridgeQuote, debouncedFromValue` injected into this hook via `useBridgeState()` hook? 

That way we can re-use this hook without having to import `useBridgeState()` into components